_data/accessibility-tests/search.yml

```diff
@@ -0,0 +1,175 @@
+title: Search
+component_status: pass
+test_items:
+# General tests
+  - summary: Color is not the only method used to convey the search button's meaning.
+    summary_additional: When using the search button, "search" text or a magnifying glass icon convey the button's meaning.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.0
+    wcag_criterion: 1.4.1
+  - summary: Search button background and text meet color contrast requirements. 
+    summary_additional: |
+      When you use ANDI color contrast analyzer on the search text button, the contrast between background and lettering is at least 4.5:1.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.0
+    wcag_criterion: 1.4.3
+  - summary: Search button background and icon meet color contrast requirements.
+    summary_additional: |
+      When you use ANDI color contrast analyzer on the search icon button, the contrast between the background and icon is at least 3:1.
+    test_status: conditional
+    test_type: general
+    version_tested: 3.7.1
+    wcag_criterion: 1.4.11
+  - summary: The width of the search button is at least 44 pixels.
+    summary_additional: |
+      When you select "inspect element" to review the button's size, you can see the total width of the button is at least 44 pixels.
```

Comment:
Resolved in [8526e0e](https://github.com/uswds/uswds-site/pull/2694/commits/8526e0e8d8004b7d7495c605360514fd0531da6b)

Reply:
I actually have a question about this one. It looks like the size of our search buttons have a minimum width of 44 pixels but the height currently comes in at 32px for the non-big variants. Does this mean we aren't meeting the criteria? Or does the area size come into effect here?

It looks like we have two issues reported to increase the tap size of the button component (#5813) and frequently used interactive elements (#5382) but not search specifically

CC: @amyleadem @amycole501  @alex-hull

Reply:
[This GH w3 resource](https://w3c.github.io/wcag/understanding/pointer-target-spacing#:~:text=The%20requirement%20is%20for%20targets,separates%20it%20from%20adjacent%20targets.) may (?) help clarify the touch size. Could we add padding on the top and bottom to meet 44px?

Reply:
@amycole501  That helps a lot actually! I didn't realize that spacing between elements adds to the touch target. In that case, the user just needs to make sure there is roughly 5px (at least) of spacing between these elements. If design has a say in anything, they should already be doing this.

Do you think it'd be worth making this a conditional test and adding a specification that the components should have proper space between other touch elements and/or shouldn't be touching any other touch elements vertically?

Reply:
YES! I think if you want to add that into the specifications or guidance that'd be very helpful. It's good to have a dev perspective on this. Some of the tests are more technical than others so your review is especially helpful on this one.

Reply:
@amycole501 @amyleadem  Updated it to the following. Might need a bit of Gherkining to polish it but what do you think of this:

> **The height and width of the search button touch target is at least 44 pixels.**
      Touch target size can be calculated by adding the height or width of the element and including the spacing units that separate it from adjacent targets.
      When you select "inspect element" to review the button's size, you can see the the button and it's spacing is at least 44 pixels wide by 44 pixels tall.

Reply:
Resolving this conversation because the initial comment has been addressed. Continuing the conversation about the updated content in this [comment thread](https://github.com/uswds/uswds-site/pull/2694#discussion_r1636695744).